frontend/css/src/styles.css

```
@@ -2,445 +2,627 @@
+    opacity: 1;
+  }
+}
 
-    .hover-lift:hover {
-        transform: translateY(-2px);
-        box-shadow: 
-            0 0 0 1px rgba(255, 255, 255, 0.06),
-            0 12px 40px rgba(0, 0, 0, 0.4),
-            0 0 60px rgba(251, 191, 36, 0.08);
-    }
+@keyframes successBoxFadeOut {
```

Stylelint (17.13.0)

[error] Expected keyframe name "successBoxFadeOut" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)